odoo/http.py

```diff
@@ -1725,7 +1725,8 @@ def _serve_db(self):
             except Exception as exc:
                 if isinstance(exc, HTTPException) and exc.code is None:
                     raise  # bubble up to odoo.http.Application.__call__
-                exc.error_response = self.registry['ir.http']._handle_error(exc)
+                if not getattr(exc, 'error_response', None):
+                    exc.error_response = self.registry['ir.http']._handle_error(exc)
```

Comment:
useless diff thanks to `werkzeug.exceptions.abort`

Reply:
good catch :)